hotxlfp/grammarparser/parser.py

```diff
@@ -154,7 +154,7 @@ def p_expression_string(self, p):
         """
         expression : STRING
         """
-        p[0] = lambda args: p[1][1:-1]
+        p[0] = lambda args, p1=p[1]: p1[1:-1]
```

Comment:
This is the main fix and I did this by inspection. I followed the pattern of the other handlers.

My take is the the reference to `p` changes underneath before this lambda actually resolves and thats why we put it as a default argument to evaluate `p[1]` as we create the lambda